src/Confluent.Kafka/Impl/SafeKafkaHandle.cs

```diff
@@ -565,6 +565,50 @@ internal Task<CommittedOffsets> CommitAsync()
         internal Task<CommittedOffsets> CommitAsync(IEnumerable<TopicPartitionOffset> offsets)
             => Task.Run(() => commitSync(offsets));
 
+        internal List<TopicPartitionError> Pause(IEnumerable<TopicPartition> partitions)
+        {
+            ThrowIfHandleClosed();
```

Comment:
it's about time the boiler plate stuff here got factored out, but this applies to other methods too, so I think that can be an exercise for a later pr.